### DISCUSSION TITLE
Pull request to datasets vs. pushing to the hub

### DISCUSSION THREAD

**Discussion by bhigy** [huggingface/datasets#3624] [Q&A] [answered]
I created a data loader for a new dataset which I pushed to the hub (bhigy/buckeye_asr). I saw that you can also contribute a dataset with a pull request to the codebase. What is the difference between the two options?

Thanks in advance for the clarification.

**Comment by mariosasko** [comment, marked as answer]
Hi! We recommend adding new datasets to the user's namespace whenever possible (to delegate maintenance to them and avoid the review process) and using Pull Requests only when the user is not the dataset owner.
